### DISCUSSION TITLE
MrTK3 Aquarium Tutorial update?

### DISCUSSION THREAD

**Discussion by Victor-x1** [microsoft/MixedRealityToolkit-Unity#11173] [General]
https://github.com/microsoft/MRTK3-iet-tutorials

Hello, I have a curiosity. The MRTK3 "Aquarium" in-editor tutorials use pre11. Is there a timeframe to when it will be updated to pre12? I have been using "Aquarium" for our computer club workshops. Thank you for all the hard work you have done!

**Comment by Zee2** [comment]
Hey Victor! That's amazing to hear! 

We'll be upgrading the tutorials to the latest MRTK3 versions over time. It does take time and we have a very limited number of people available to help out, so stay tuned!
